frontend/packages/dev-console/src/components/import/serverless-function/AddServerlessFunctionForm.tsx

```diff
@@ -38,6 +41,9 @@ enum SupportedRuntime {
   quarkus = 'java',
 }
 
+export const SERVERLESS_FUNCTION_DOCS_URL =
+  'https://docs.openshift.com/serverless/latest/functions/serverless-functions-getting-started.html';
```

Comment:
better to use `getDocumentationURL` util to get the doc URL.

Reply:
The `getDocumentationURL` afaik, is for the links that are present in the OKD docs, currently the Serverless docs are not in the OKD docs.